### PR TITLE
Fix armel tests build

### PR DESCRIPTION
**Pull request by gbalykov**
After this PR System.Private.CoreLib.dll is copied to bin dir if `-skipcrossgen` is passed to build.sh, and build-test.sh successfully finishes for `armel` cross build. 

Also, cross build is fixed if `ROOTFS_DIR` is not passed explicitly.

cc @alpencolt @alexander-aksenov

### CODE REVIEW

File: build-test.sh [hunk 1]
```diff
@@ -317,7 +317,10 @@ build_Tests()
     fi
 
     if [ $__SkipNative != 1 ]; then
+        export CROSSCOMPILEREAL=${CROSSCOMPILE}
```
Comment (jashook) [inline build-test.sh]: What are these being used for?

File: build.sh [hunk 2]
```diff
@@ -435,13 +435,19 @@ build_CoreLib()
...
     fi
 
+    local __CoreLibILDir=$__BinDir/IL
+
     if [ $__SkipCrossgen == 1 ]; then
         echo "Skipping generating native image"
+
+        if [ $__CrossBuild == 1 ]; then
+            # Crossgen not performed, so treat the IL version as the final version
+            cp $__CoreLibILDir/System.Private.CoreLib.dll $__BinDir/System.Private.CoreLib.dll
```
Comment (jashook) [inline build.sh]: This is a somewhat scary change, as it is easy to forget that this SPC is not crossgened for one specific target. @RussKeldorph do you have an opinion.

File: build.sh [hunk 3]
```diff
@@ -435,13 +435,19 @@ build_CoreLib()
         exit 1
     fi
 
+    local __CoreLibILDir=$__BinDir/IL
+
     if [ $__SkipCrossgen == 1 ]; then
         echo "Skipping generating native image"
+
+        if [ $__CrossBuild == 1 ]; then
```
Comment (jashook) [inline build.sh]: This should be changed to armel specifically, we do not want to use the IL SPC for arm64 or armhf. It is unclear to me if we want the IL SPC for armel.

File: build.sh [hunk 4]
```diff
@@ -1011,6 +1017,14 @@ if [ $__CrossBuild == 1 ]; then
 fi
 __CrossGenCoreLibLog="$__LogsDir/CrossgenCoreLib_$__BuildOS.$__BuildArch.$__BuildType.log"
 
+# Configure environment if we are doing a cross compile.
+if [ $__CrossBuild == 1 ]; then
+    export CROSSCOMPILE=1
```
Comment (jashook) [inline build.sh]: What is the env variable `CROSSCOMPILE` being used for?

File: build.sh (hunk 2 above)
Comment (gbalykov) [reply build.sh]: This behavior is same to the one before #24285, i.e. if `-skipcrossgen` was passed, SPC.dll was copied to bin dir in `MovePostBuildBinaries`

Comment (jashook) [reply build.sh]: Ah I see, I missed the skipcrossgen flag. Thanks, makes much more sense

File: build.sh (hunk 4 above)
Comment (gbalykov) [reply build.sh]: As far as I can tell, it's used to set some platform specific defines

File: build-test.sh (hunk 1 above)
Comment (gbalykov) [reply build-test.sh]: `CROSSCOMPILEREAL` is used to just temporarily save value of `CROSSCOMPILE`. Logic here is now the same as in build.sh, where `CROSSCOMPILE` is set to 0 before build_native

File: build.sh (hunk 4 above)
Comment (jashook) [reply build.sh]: This seems fine

File: build-test.sh (hunk 1 above)
Comment (jashook) [reply build-test.sh]: I am confused why we would unset crosscompile here for the native build. It seems like that is where it is actually needed.

Comment (gbalykov) [reply build-test.sh]: You were right, this was an incorrect change. Correct fix was to properly setup portable build parameter. I've updated PR.

File: build-test.sh [hunk 5]
```diff
@@ -948,6 +948,15 @@ if [[ $__ClangMajorVersion == 0 && $__ClangMinorVersion == 0 ]]; then
     fi
 fi
 
+if [[ "$__BuildArch" == "armel" ]]; then
```
Comment (jashook) [inline build-test.sh]: Can you hoist this decision out of build-test and put it in init-distro-rid.sh

Comment (gbalykov) [reply build-test.sh]: I've updated `build.sh` and `build-test.sh`. Also, I've fixed armel build for priority1 tests.

**Comment by gbalykov** [comment]
@jashook could you, please, take a look

**Pull request merged**